### PR TITLE
Added license badge

### PR DESCRIPTION
**Pull request by LucasTJerez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
-# project-proposals-f2020
-Project Proposals for Fall 2020
+# WEAT
```
Comment (timkpaine) [inline README.md]: dont modify readme, modify <uni>.md

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LucasTJerez) [reply README.md]: I thought that we'd need a README to display on github. Can I just delete this whole file then?

File: LICENSE [hunk 2]
```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```
Comment (timkpaine) [inline LICENSE]: don't remove my file

File: README.md [hunk 3]
```diff
@@ -1,5 +0,0 @@
-# project-proposals-f2020
```
Comment (timkpaine) [inline README.md]: don't remove my file

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -3,3 +3,4 @@ Project Proposals for Fall 2020
 
 Put your project proposal in this repo in the form: `<uni>.md`.
 E.g. for me, i would make a file called `tkp2108.md`.
+
```
Comment (timkpaine) [inline README.md]: don't change my readme.

File: tkp2108.md [hunk 5]
```diff
@@ -1,7 +0,0 @@
-Project Proposal
```
Comment (timkpaine) [inline tkp2108.md]: don't delete my file.

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Pull request merged**